package.json

```diff
@@ -2,6 +2,7 @@
   "name": "workshop",
   "version": "1.0.0",
   "description": "",
+  "type": "module",
```

Comment:
this breaks eslint, easiest fix would be renaming its configuration file to .eslintrc.cjs